go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/hashicorp/terraform-plugin-docs v0.12.0
 	github.com/hashicorp/terraform-plugin-log v0.4.1
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.17.0
-	github.com/jianyuan/go-sentry/v2 v2.0.0-20220624071746-b886357dc280
```

Comment:
It is better to use `replace` instead of changing import paths everywhere. It will be challenging to keep it updated with upstream; now, we will have conflict in all files.

Reply:
I saw that I also changed my replace to imports: https://github.com/Canva/terraform-provider-sentry/commit/b852f55dadbf4f55b953f614ad9315ce283766fa
If I'm not wrong, it had something to do with the idea that I wanted to use `git submodule` for the go-sentry component. If that is the only issue, we can remove the submodule.

Reply:
This is so much better. Thanks for the suggestion. I fixed the imports and removed the git submodule in https://github.com/Canva/terraform-provider-sentry/pull/28/commits/96e6b086dc5f16f238fc5c56632e8cdc667d8062